### PR TITLE
168/Integrate curd operations using node database for angular app

### PR DESCRIPTION
**Pull request by rajat-vrma**
Issue: #168

### CODE REVIEW

**Review by manjeetbeniwalinnostax — APPROVED** [review]

**Comment by manjeetbeniwalinnostax** [comment]
Will resolve comments in followup PR

File: src/angularApiTemplates/services/services.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  errorMgmt(error: HttpErrorResponse) {
+    let errorMessage = '';
+    if (error.error instanceof ErrorEvent) {
+      // Get client-side error
+      errorMessage = error.error.message;
+    } else {
+      // Get server-side error
+      errorMessage = `Error Code: ${error.status}\nMessage: ${error.message}`;
+    }
+    console.log(errorMessage);
```
Comment (Chaman-Raghav) [inline src/angularApiTemplates/services/services.ts:48]: console.log(errorMessage);
=>
console.error(errorMessage);
maybe we can use this

File: src/angularApiTemplates/services/services.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    return throwError(() => {
+      return errorMessage;
+    });
```
Comment (Chaman-Raghav) [inline src/angularApiTemplates/services/services.ts:51]: no need of curly brace in whole single line function

**Review by Chaman-Raghav — COMMENTED** [review]
Approved with minor suggestions

File: src/angularApiTemplates/services/services.ts (hunk 2 above)
Comment (rajat-vrma) [reply src/angularApiTemplates/services/services.ts:51]: Updated to:
return throwError(() => errorMessage);

**Pull request merged**